ide/languages.toml/nbproject/project.properties

```diff
@@ -14,6 +14,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-javac.source=1.8
+javac.source=11
```

Comment:
for consistency reasons please set source **and** target:
```
javac.source=11
javac.target=11
```
at some point we will search/replace over that and use only `javac.release=11` where possible.